new: more adapter housekeeping

aMahanna opened the pull request:
* add debug logs on individual node/edge level
* update README
* package config cleanup
* new `__insert_adb_docs` method
* clean up `arangodb_to_dgl` and `dgl_to_arangodb`
* clean up `etypes_to_edefinitions`

aMahanna committed:
new: debug log on individual node/edge levels

aMahanna committed:
Update README.md

cw00dw0rd commented:
Discussed in person:
- Investigate python-arango logging
- Use logging for better error handling as well

aMahanna committed:
cleanup

aMahanna committed:
config cleanup

aMahanna committed:
fix: typo

aMahanna committed:
new: __insert_adb_docs

also: cleans up node/edge insertion in ArangoDB to DGL

aMahanna committed:
Update setup.cfg

aMahanna committed:
Update conftest.py

aMahanna committed:
cleanup: __fetch_adb_docs

was complicating things for no reason

aMahanna committed:
cleanup & fix: etypes_to_edefinitions

also:

aMahanna committed:
fix: mypy

aMahanna committed:
fix: metagraph example

aMahanna committed:
new: remove ValueError exception

I was under the impression that working with an ArangoDB edge collection that had multiple "from" and "to" vertex types was not possible to convert to DGL. This commit allows for it

aMahanna committed:
fix: black

aMahanna committed:
fix: tests

aMahanna committed:
Update adapter.py

aMahanna committed:
fix: black

aMahanna committed:
cleanup

aMahanna committed:
Update conftest.py

aMahanna committed:
fix: isort
